WindowsServerDocs/storage/storage-spaces/Cluster-Sets.md

```diff
@@ -97,7 +97,9 @@ The following considerations apply to an Infrastructure SOFS role:
 
 1.    There can be at most only one Infrastructure SOFS cluster role on a Failover Cluster. Infrastructure SOFS role is created by specifying the "**-Infrastructure**" switch parameter to the **Add-ClusterScaleOutFileServerRole** cmdlet.  For example:
 
-        Add-ClusterScaleoutFileServerRole -Name "my_infra_sofs_name" -Infrastructure
+    ```PowerShell
+    Add-ClusterScaleoutFileServerRole -Name "my_infra_sofs_name" -Infrastructure
+    ```
```

Comment:
This code snippet does not render properly in the Rich View mode. I can't see anything wrong with it, so perhaps it has something to do with the code around it.

Reply:
@skycommand Thank you for your review. I added new commit to fix it.

Reply:
You may want to minimize this conversation by marking it as "Resolved" (if available).